Makefile.am

```diff
@@ -60,11 +60,33 @@ AM_CPPFLAGS = \
 # Utility library
 noinst_LTLIBRARIES += libeos-updater-util/libeos-updater-util-@EUU_API_VERSION@.la
 
+enum_header_files = \
+	libeos-updater-util/flatpak.h \
+	eos-updater-flatpak-installer/installer.h \
+	$(NULL)
+enum_template_h_file = libeos-updater-util/eos-updater-enums.h.in
+enum_template_c_file = libeos-updater-util/eos-updater-enums.c.in
```

Comment:
Drop the `eos-updater-` prefix from the files, and call them `libeos-updater-util/enums.[ch]{,.in}` instead. I’m trying to move to a model where filenames don’t have a namespace prefix, because the directory they’re in provides their namespace. It makes tab-completion easier (and reduces typing for the filenames).